mgpath/path.go

```diff
@@ -50,3 +51,29 @@ func Tools() string {
 	}
 	return mgToolsPath
 }
+
+func ChangeWorkDir(path string) func() {
+	cwd := FromWorkDir(".")
+	if err := os.Chdir(path); err != nil {
+		panic(err)
+	}
+	return func() {
+		if err := os.Chdir(cwd); err != nil {
+			panic(err)
+		}
+	}
+}
+
+func FileExtensionsIn(ext, path string) ([]string, error) {
```

Comment:
Wondering if using the word `find` would be good since it sort of replaces the unix find command. `FindFilesWithExtension(".", ".proto")` ~ `find . -name '*.proto'`?

Reply:
I like it!

Reply:
Fixed